mantidproject/mantid
# REF_M reduction update

IN PROGRESS

Write a workflow algorithm to reduction REF_M data, adapted from the REF_L algorithm.

**To test:**
- Inspect the code.
- Make sure the system tests pass.

There is no GitHub issue for this PR. All the information is here.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
